### PR TITLE
add MD4 and MD5 hashes

### PR DESCRIPTION
**Pull request by celeduc**
Resolve [multihash#71](https://github.com/multiformats/multihash/issues/71) by adding MD4 and MD5 hashes.  See [multihash#76](https://github.com/multiformats/multihash/pull/76).

cc @lgierth @diasdavid @whyrusleeping

### CODE REVIEW

**Comment by Kubuxu** [comment]
What do you think about moving them into two byte range? They should not be used with content addressing where space is the key.

cc @whyrusleeping @jbenet @dignifiedquire

**Comment by celeduc** [comment]
If there is a proposal for exactly where to put these in the two-byte range, then I'm all for it, and I'll make all of the changes necessary.

Also, I'll remove the file from [multihash](https://github.com/multiformats/multihash/blob/master/hashtable.csv) where these are defined, since there should be only one place where these are maintained.

**Comment by diasdavid** [comment]
> They should not be used with content addressing where space is the key.

@Kubuxu I'm not sure if I understand what you meant here

**Comment by Kubuxu** [comment]
With content addressing you use multihash repeated many times which is linked and so on. This means that one additional byte makes a difference, especially with transfers and so on. IMO as MD5 and MD4 are depracated, it makes sense to save some space in one byte range (0-127) and move it to two byte range.

**Comment by celeduc** [comment]
@Kubuxu Could you please suggest values that meet with your requirements so this can be finished?

**Comment by Kubuxu** [comment]
How about `0xd3` and `0xd4`,

**Comment by celeduc** [comment]
Changed as per @Kubuxu above.

File: table.csv [hunk 1]
```diff
@@ -35,6 +35,8 @@ multiaddr,          ,                         0x32
 multibase,          ,                         0x33
 
 multihashes,,
+md4,                ,                         0xd3
+md5,                ,                         0xd4
```
Comment (diasdavid) [inline table.csv]: Well, if we are going to arbitrarily pick a number, better make it `0xd4` and `0xd5`. Sounds good @Kubuxu ?

Comment (Kubuxu) [reply table.csv]: SGWM

Comment (celeduc) [reply table.csv]: Done and done.

**Comment by celeduc** [comment]
Ready for merge!

**Pull request merged**